doc/source/contributing.rst

```diff
@@ -170,7 +170,7 @@ We'll now kick off a three-step process:
 .. code-block:: none
 
    # Create and activate the build environment
-   conda env create -f ci/environment-dev.yaml
+   conda env create
```

Comment:
this is not a valid command, you need to name the env (or use a -p)

Reply:
I don't see it in the conda docs, but if no `-f` or `-p` is provided, `conda env create` (or `conda env upate`) will use the file `environment.yml` which is the standard location for it (note the `.yml` instead of `.yaml` too).

As we now have the `environment.yml` in the stadard location, I think just `conda env create` is better (easier to remember and faster to type).

Reply:
I would still favor using the explicit ``conda env create -f environment.yml``. Yes, it is more typing, but IMO it's much more explicit and clear in what is happening. Otherwise you need to know that the standard file is used if none is specified, which even Jeff and me didn't know.